From b8e8aa87f43c12ad564426b3359f593305cd45a1 Mon Sep 17 00:00:00 2001
From: Roland Knall <rknall@gmail.com>
Date: Wed, 25 Apr 2018 10:28:30 +0200
Subject: [PATCH] Qt: Fix various missing header includes

Qt 5.11 seems to have changed the include dependencies, so adding those, that are missing

Change-Id: I2b0482f7554467d6981be65bfd3fea1a3e118976
Reviewed-on: https://code.wireshark.org/review/27145
Petri-Dish: Roland Knall <rknall@gmail.com>
Tested-by: Petri Dish Buildbot
Reviewed-by: Roland Knall <rknall@gmail.com>
---
 ui/qt/packet_format_group_box.cpp | 1 +
 ui/qt/time_shift_dialog.cpp       | 1 +
 ui/qt/wireless_frame.cpp          | 1 +
 3 files changed, 3 insertions(+)

diff --git a/ui/qt/packet_format_group_box.cpp b/ui/qt/packet_format_group_box.cpp
index a80a71aaaddfce2798ececcbb27a4d9e75382c1b..3c0fff28aa4cd91d344f2e70db1e66123bc17f41 100644
--- a/ui/qt/packet_format_group_box.cpp
+++ b/ui/qt/packet_format_group_box.cpp
@@ -10,6 +10,7 @@
 #include <ui_packet_format_group_box.h>
 
 #include <QStyle>
+#include <QStyleOption>
 
 PacketFormatGroupBox::PacketFormatGroupBox(QWidget *parent) :
     QGroupBox(parent),
diff --git a/ui/qt/time_shift_dialog.cpp b/ui/qt/time_shift_dialog.cpp
index 93882c09a377a5630e8bb09b4e80d7f6a84f7293..8460db2e966a0f572478de7430b8b63f4473dd16 100644
--- a/ui/qt/time_shift_dialog.cpp
+++ b/ui/qt/time_shift_dialog.cpp
@@ -14,6 +14,7 @@
 #include <ui/time_shift.h>
 #include <ui/qt/utils/tango_colors.h>
 
+#include <QStyleOption>
 
 TimeShiftDialog::TimeShiftDialog(QWidget *parent, capture_file *cf) :
     QDialog(parent),
diff --git a/ui/qt/wireless_frame.cpp b/ui/qt/wireless_frame.cpp
index bd9076abcf4a5d7b6fd73d2bc0b63b4fff8c256a..46fb2dd2f3054f66380239106fb2e27058196447 100644
--- a/ui/qt/wireless_frame.cpp
+++ b/ui/qt/wireless_frame.cpp
@@ -23,6 +23,7 @@
 #include <wsutil/frequency-utils.h>
 
 #include <QProcess>
+#include <QAbstractItemView>
 
 // To do:
 // - Disable or hide invalid channel types.
